package/MDAnalysis/analysis/contacts.py

```diff
@@ -373,6 +373,8 @@ class Contacts(AnalysisBase):
     .. versionchanged:: 1.0.0
        ``save()`` method has been removed. Use ``np.savetxt()`` on
        :attr:`Contacts.timeseries` instead.
+    .. versionchanged:: 2.0.0
```

Comment:
Thanks @ss62171, just one final thing -- this should be for version 1.0.0 which hasn't been released yet.